scrapers/USA/WA/index.js

```diff
@@ -93,8 +93,16 @@ const scraper = {
       const $trs = $table.find('tbody > tr');
       $trs.each((index, tr) => {
         const $tr = $(tr);
-        const cases = parse.number($tr.find('td:nth-child(2)').text());
-        const deaths = parse.number($tr.find('td:last-child').text());
+        let cases = parse.number($tr.find('td:nth-child(2)').text());
+        let deaths = parse.number($tr.find('td:last-child').text());
```

Comment:
So with this code, if the text changes to `something`, this will return zero. It's possible that, if the text is truly a non-empty and non-number thing, we want an error.

In that case, this should still work for empty cells, but should still explode if the cell contains non number data:

```suggestion
        let deaths = parse.number(parse.string($tr.find('td:last-child').text()) || 0);
```
Thoughts? Silent failures are scary.